protocols/raft/src/main/java/io/atomix/protocols/raft/roles/LeaderAppender.java

```diff
@@ -387,7 +394,7 @@ protected void handleAppendResponseError(
     // If we've received a greater term, update the term and transition back to follower.
     if (response.term() > raft.getTerm()) {
       log.info(
-          "Received higher term ({} > {}) from {}",
+          "Received error append response with higher term ({} > {}) from {}, stepping down",
```

Comment:
Nit: (same as above) It's not "stepping down" but transitioning to follower because there is a new leader.